pty/src/win/conpty.rs

```diff
@@ -1,5 +1,5 @@
 use crate::cmdbuilder::CommandBuilder;
-use crate::win::psuedocon::PsuedoCon;
+use crate::win::pseudocon::PsuedoCon;
```

Comment:
Randomly passing by, but shouldn't you rename the `PsuedoCon` struct as well while you are at it ?

Reply:
thanks, missed that one, addressed with the latest update